src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const firstNumber = (a) => {
+    return function secondNumber(b) {
```

Comment:
You don't need to name functions if you return them right away (checklist point 2)